apps/roam/src/styles/styles.css

```diff
@@ -114,6 +114,17 @@
   border-bottom: 1px solid rgba(16, 22, 26, 0.15);
 }
 
+.bp3-callout.invert-icon .bp3-icon {
```

Comment:
nit: unneeded.  `intent: primary` on the `<Callout>` should get you most of the way there.

See `LabelDialog` example and Callout docs

https://blueprintjs.com/docs/versions/3/#core/components/callout
<img width="1714" height="915" alt="Image" src="https://github.com/user-attachments/assets/c9bfeb35-6386-4db1-95e0-bc78f9da5156" />


LabelDialog
<img width="1472" height="709" alt="Image" src="https://github.com/user-attachments/assets/503f3e1b-1ee7-4e89-8190-0f16227c5630" />